src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -722,8 +722,8 @@ void Graph::AllocateWithReuse() {
 
     std::vector<MemorySolver::Box> definedBoxes;
     std::vector<MemorySolver::Box> undefinedBoxes;
-    for (int i = 0; i < edge_clusters.size(); i++) {
-        MemorySolver::Box box = { std::numeric_limits<int>::max(), 0, 0, i };
+    for (size_t i = 0; i < edge_clusters.size(); i++) {
+        MemorySolver::Box box = {std::numeric_limits<int>::max(), 0, 0, static_cast<int64_t>(i)};
```

Comment:
MemorySolver::Box's last element is **int64_t id**, if change to size_t, it will arise other warning.
And box.id also used in other place, so this update has minimal changes.